.github/workflows/lint.yml

```diff
@@ -26,10 +26,10 @@ jobs:
         architecture: x64
 
     - name: Set up linters
-      uses: world-federation-of-advertisers/actions/setup-linters@v1.3.0
+      uses: world-federation-of-advertisers/actions/setup-linters@v1.4.0
```

Comment:
Why are these broken into two? 

(I know, it's not really part of the PR but I'm interested what the reasoning is)

Reply:
> fyi, for future PRs like this, it's easier to review if you break this into two commits, one for your manual changes (update to GH Actions version, etc) and one for the ktfmt changes.

You want the enforcement to be lock-step with the one-time global formatting. This way you can say anything after this PR is merged will be formatted.

Reply:
> Why are these broken into two?

They're separate actions within the same repo that have separate responsibilities and therefore separate types. `setup-linters` is a [JavaScript action](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action) that installs the linters. `lint` is a [composite run steps action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-run-steps-action) that runs a script which lints files changed by a PR.

Reply:
👍  thx!